posthog/src/main/java/com/posthog/PostHogInterface.kt

```diff
@@ -2,10 +2,7 @@ package com.posthog
 
 import java.util.UUID
 
-/**
- * The PostHog SDK entry point
- */
-public interface PostHogInterface {
+public interface PostHogCoreInterface {
```

Comment:
lets create new files for new classes/interfaces (usually we don't do multiple things in one class in java world)